python/tests/snapshots/quantum-conditional-1.ll

```diff
@@ -11,30 +11,30 @@ alloca_block:
   call void @__quantum__qis__phasedx__body(double 0x3FF921FB54442D18, double 0xBFF921FB54442D18, %Qubit* null)
   call void @__quantum__qis__rz__body(double 0x400921FB54442D18, %Qubit* null)
   call void @__quantum__qis__phasedx__body(double 0xBFF921FB54442D18, double 0x3FF921FB54442D18, %Qubit* inttoptr (i64 1 to %Qubit*))
-  call void @__quantum__qis__zzmax__body(%Qubit* null, %Qubit* inttoptr (i64 1 to %Qubit*))
+  call void @__quantum__qis__rzz__body(%Qubit* null, %Qubit* inttoptr (i64 1 to %Qubit*))
```

Comment:
is zzmax and rzz the same?

Reply:
No, `rzz` has a phase parameter; this looks wrong.

Reply:
I though the same. And I would assume there is no option to just have no parameter and is it assumed this is the full version?

Reply:
(will look into this now and try to fix it)

Reply:
solved in [475d688](https://github.com/CQCL/hugr-qir/pull/71/commits/475d68825964fc02e12fa44cc17922e8f539b223)